packages/lib/modules/pool/actions/LiquidityActionHelpers.ts

```diff
@@ -291,6 +291,22 @@ export function supportsProportionalAddLiquidityKind(pool: Pool): boolean {
   return true
 }
 
+// v2 Pools with WeightedPool2Tokens pool types do not support AddLiquidityKind.Proportional in the SDK
+// TODO: get this data from the API
+export function hasWeightedPool2Tokens(pool: Pool): boolean {
```

Comment:
Do you know for sure that it's limited to 2 token pools? My understanding is that we had a couple of versions of weighted pool and all could have more than 2 tokens.

Reply:
I'm talking to the SDK/SC teams to confirm. I believe we can merge and create another PR when we have that info.